proxies/nodejs/entityTypes.ts

```diff
@@ -1,6 +1,36 @@
 export enum EntityTypes {
-  user = "User",
-  variable = "Variable",
-  feature = "Feature",
-  object = "Object"
+  user = 'User',
+  variable = 'Variable',
+  feature = 'Feature',
+  object = 'Object',
+}
+
+export const getEntityModelFromType = (value: string): EntityTypes => {
```

Comment:
if these functions don't get called individually without the other you can probably just combine them to return an `Entity` object that has both model and name